### PR TITLE
github actions: Update black to 20.8b1

### PR DESCRIPTION
**Pull request by bgurney-rh**
Related: stratis-storage/project#269

### CODE REVIEW

File: tests/whitebox/_misc.py [hunk 1]
```diff
@@ -67,11 +67,13 @@ def setup(self):
             raise RuntimeError(
                 "STRATISD environment variable must be set to absolute path of stratisd executable"
             )
-        self._stratisd = subprocess.Popen(  # pylint: disable=attribute-defined-outside-init
-            [os.path.join(stratisd_var), "--sim"],
-            stdout=subprocess.PIPE,
-            stderr=subprocess.PIPE,
-            universal_newlines=True,
+        self._stratisd = (
+            subprocess.Popen(  # pylint: disable=attribute-defined-outside-init
```
Comment (mulkieran) [inline tests/whitebox/_misc.py:71]: Put this disable after the open paren on the line above and pylint will stop compalining.

**Review by mulkieran — CHANGES_REQUESTED** [review]
The only request I have is that where reformatting of header comments has occured, that you align the top header comment with all the rest.

File: src/stratis_cli/_errors.py [hunk 2]
```diff
@@ -99,15 +99,15 @@ class StratisCliPartialChangeError(StratisCliUserError):
     """
 
     def __init__(self, command, changed_resources, unchanged_resources):
-        """ Initializer.
+        """Initializer.
```
Comment (mulkieran) [inline src/stratis_cli/_errors.py:102]: Like here is an example. Just move ```Initializer``` down a line so it's in line with those below.

File: tests/whitebox/_misc.py (hunk 1 above)
Comment (bgurney-rh) [reply tests/whitebox/_misc.py]: Resolved.

File: src/stratis_cli/_errors.py (hunk 2 above)
Comment (bgurney-rh) [reply src/stratis_cli/_errors.py]: Resolved.

File: src/stratis_cli/_errors.py [hunk 3]
```diff
@@ -347,11 +353,11 @@ class StratisCliEnginePropertyError(StratisCliRuntimeError):
     """
 
     def __init__(self, prop_name, message):
-        """ Initializer.
+        """Initializer.
```
Comment (mulkieran) [inline src/stratis_cli/_errors.py:356]: Missed one!

**Review by mulkieran — CHANGES_REQUESTED** [review]

File: src/stratis_cli/_errors.py (hunk 3 above)
Comment (bgurney-rh) [reply src/stratis_cli/_errors.py]: Resolved.

**Pull request merged**